doc/content/source/vectorpostprocessors/Fragility.md

```diff
@@ -12,6 +12,22 @@ demand distribution and capacity distribution and calculate the conditional prob
 in each bin, and 3) fit a lognormal distribution in the conditional probabilities to calculate a
 fragility. This kind of fragility is also referred to as 'enhanced fragility'.
 
+Fitting a lognormal fragility function given analysis results requires optimization to find
+the parameters median and lognormal standard deviation. Two optimization methods are currently available: brute force
+and Stochastic Gradient Descent (SGD). Brute force finds that combination of parameters
+ which maximizes the likelihood by looping over all alternative combinations. SGD finds the parameter
+ combination by running a Newton-Raphson type algorithm with random seed values. SGD also requires the additional parameters:
+
+1. tolerance (To declare convergence of the algorithm. 1e-03 can be optimal.)
+2. gamma (Controls the step size of the algorithm. 0.001 can be optimal.)
+3. numrnd (Number of random seed values. 1000 can be optimal.)
+4. seed (Seed for the random number generator. Currently set to 1028).
+
+Of course, both algorithms have their pros and cons. For example, brute force takes orders of magnitude more time to run. SGD
+does not always guarantee precise results.
```

Comment:
Please say that 'SGD may not guarantee the best fragility fit and users should try to change the SGD parameters to ensure that the SGD results do not change significantly.'